resources/qml/PrintSetupHeaderButton.qml

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) 2022 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+// Button component to be used within the
```

Comment:
Within the...?

Reply:
https://github.com/Ultimaker/Cura/pull/11336/commits/f94813a98e26b2825ccdc299743f764713c9d5c1